docs/pages/build/building-from-github.mdx

```diff
@@ -12,6 +12,23 @@ This guide explains how to trigger builds directly from your GitHub repository u
 
 ## Prerequisites
 
+### Set the `image` field in your eas.json
+
+For the build profiles you want to use with GitHub, you need to specify the image to use for the
+build. This is done in the **eas.json** file in your project's root directory. See [the eas.json reference](/eas/json/#image) to learn more.
```

Comment:
```suggestion
For the build profiles you want to use with GitHub, specify an [`image`](/eas/json/#image) to use for the native platform in **eas.json**.

Use the `latest` image if your project's configuration does not rely on a specific [build image](/build-reference/infrastructure/). For example:
```

- Let's make the first line concise and also link to the specific part ("what") of the text instead of mentioning to see the eas.json schema
- Let's also add a description now what images should use in default cases and link to build server infrastructure guide.